library-extensions/src/main/java/com/mikepenz/fastadapter_extensions/drag/SimpleDragCallback.java

```diff
@@ -63,6 +66,17 @@ public boolean isLongPressDragEnabled() {
 
     @Override
     public boolean onMove(RecyclerView recyclerView, RecyclerView.ViewHolder viewHolder, RecyclerView.ViewHolder target) {
+
+        // remember the from/to positions
+        if (target.itemView.getTag() instanceof IDraggable) {
+            if (((IDraggable) target.itemView.getTag()).isDraggable()) {
+                if (mFrom == null) {
```

Comment:
What do you mean? The `mFrom == null` check? This is necessary because we want to save the very first from position only and don't want to overwrite this value while dragging around the item